iocore/cache/CacheHosting.cc

```diff
@@ -656,6 +656,7 @@ ConfigVolumes::BuildListFromString(char *config_file_path, char *file_buf)
     CacheType scheme  = CACHE_NONE_TYPE;
     int size          = 0;
     int in_percent    = 0;
+    bool ramcache     = true;
```

Comment:
I'd really prefer a slightly different name, to better distinguish this from "ram_cache", i.e. member that is an actual data structure and not a flag. I use the suffix "\_p" for "predicate" in such circumstances, although a prefix of "f_" is used in various places. Or "ramcache_enabled". Then you have things like
```
if (gvol[i]->cache_vol->ramcache_p) {
```
or
```
if (gvol[i]->cache_vol->ramcache_enabled) {
```

Reply:
Agreed, renamed all those to ramcache_enabled